server/utils/plugins.ts

```diff
@@ -140,7 +140,7 @@ export const OpenSearchObservabilityPlugin = (client: any, config: any, componen
   });
 
   observability.getJobStatus = createAction(client, components, {
-    endpoint: `${URI.ASYNC_QUERY}/<%=queryId%>`,
```

Comment:
Think we don't need to add the `/<%=queryId%>` here. For some reason, when I tried it with it included, it would attach two instances of `/<%=queryId%>`. May need to change the other actions as well.